### ISSUE TITLE
Looper doesn't respect some settings

### ISSUE THREAD

**Issue by h0rn3z0r** [taburineagle/MPC-HC-Looper#2]
@taburineagle
Hello, I have an issue with "Save current Loop Mode button setting as default" setting - this tool simply ignores it.
Also, there is a problem with launching two instances of Looper, it asks me to enable "Player > Open a new player for each media file played" radio button even if it is already chosen.

<details>
 <summary>My MPCLooper.ini file</summary>

```ini
[System]
MPCEXE=D:\Programs\MPC-HC\mpc-hc64.exe
[StartPos]
startPositionL=1432
startPositionT=244
startPositionW=448
startPositionH=552
[Prefs]
loopButtonMode=Shuffle Mode
alwaysOnTop=0
dockMode=OFF
MPCConfirm=1
allowMultipleInstances=1

```
</details>

Versions:
MPC-HC: 1.8.4
Looper: 02.11.18
Windows 10

**Comment by taburineagle** [comment]
@h0rn3z0r - Hello!  The "Save current Loop Mode button setting as default" setting at the moment is limited to "Loop Mode" and "OFF", and it also only applies to the very first time Looper opens.  Are you referring to wanting either "Shuffle" or "Random" modes enabled right when you open a .looper file instead of jumping to "Loop Mode" by default?  At the moment, Looper always loads new files in "Loop Mode" regardless of the setting above (as that's limited to just the first time Looper opens) - please let me know on that one.

Also, on my computer (Win7, MPC-HC 1.7.10, Looper [_in development, but functionally similar to 02.11.18]_), the multiple instances feature works.  You have to make sure you have the right settings set on both Looper and MPC-HC -

In Looper, go to the Preferences and click "Allow multiple instances of Looper"... (which you did)

In MPC-HC, go to the Preferences (View menu, Options), click on "Player" at the top left, and then make sure "Open a new player for each media file played" is selected, then click Apply, and OK, and quit MPC-HC.

![open file](https://user-images.githubusercontent.com/18430526/52527066-98ae8e00-2c90-11e9-8215-84e606174c03.PNG)

With both of those preferences selected, in both Looper and MPC-HC, multiple instances _should_ work...

**Comment by h0rn3z0r** [comment]
@taburineagle 
Thanks for your answer, and it is really cool that you continued developement of this useful tool.
Okay, seems like I misunderstood the purpose of this setting. I totally agree that Looper should start in Loop mode when you create a new file, but if you open an existing playlist and only first file is playing again and again it makes no sense. It would be nice to open .looper files in shuffle/playlist mode by default without need to change mode every time, though it's more a suggestion than a bug report.

Regarding the second question, problem was caused by "Store settings in .ini file" MPC setting which I am using:
![mpc](https://user-images.githubusercontent.com/8987403/52527556-70984c80-2cdb-11e9-9afd-8c485b6db114.png)
Here you are checking user registry not taking into consideration that file:
https://github.com/taburineagle/MPC-HC-Looper/blob/f6d7cd3ddb4b44fefaef3a56c327a72cc9947786/Media%20Player%20Classic%20Looper.au3#L130
I am not sure how many people use ini settings file, propbably it's not critical. As a temporary fix I changed that registry key and problem completely gone!

**Comment by taburineagle** [comment]
@h0rn3z0r - For the first one, definitely duly noted :)  I've never had a need for a feature like that before _myself_, but I could see where it would come in handy.  Thanks for the suggestion, I'll see what I can do on that front for the next version!

For the 2nd one - ah, yes, the INI file!  I remember when I originally wrote that feature in, I wondered how many people used the INI file instead of the registry, and I went ahead and used the registry for the check.  I should write in a redundancy check to see if that option is checked in the registry (Store settings in .ini file) and then see what the value is in the .INI file as well.  I've never used the .INI file myself though, is it always stored in the same path as MPC-HC's executable file?

**Comment by h0rn3z0r** [comment]
@taburineagle 
Yes. there is a mpc-hc64.ini file right next to mpc-hc64.exe (probably mpc-hc.ini for 32-bit version) which just duplicates all registry keys.

**Comment by taburineagle** [comment]
@h0rn3z0r - Got you.  I coded in a redundancy check that checks all 3 sources for preferences (the 32-bit INI, the 64-bit INI and the registry) to see if that flag is set in any of them.  I wanted to force the INI check if "Store settings in .ini file" was set in the preferences, but for some reason, I don't have that option (I believe it's due to folder permissions) and I can't find that value in the registry, so I coded for all 3.  Tested with stand-alone versions of 32-bit and 64-bit MPC-HC (both 1.17.13), and Looper now works as it should _with_ the INI files set appropriately...
